apps/studio/components/interfaces/UnifiedLogs/components/Columns.tsx

```diff
@@ -39,9 +39,9 @@ export const COLUMNS: ColumnDef<ColumnSchema>[] = [
     },
     filterFn: (row, columnId, filterValue) => true,
     enableResizing: false,
-    size: 190,
-    minSize: 190,
-    maxSize: 190,
+    size: 130,
```

Comment:
hang on, need to check

Reply:
the column size changes were mainly to have the columns a bit more fitted for a laptop viewport
otherwise, the data table would be horizontally scrollable from the start for a laptop viewport and the "load more" button + Empty state texts like "No rows" or "Fetching rows" will be off center